templates/html_report/composer.php

```diff
@@ -0,0 +1,100 @@
+<?php
+require __DIR__ . '/_header.php'; ?>
+
+<?php
+$packages = isset($project['composer']['packages']) ? $project['composer']['packages'] : [];
+usort($packages, function ($a, $b) {
+    return strcmp($a->name, $b->name);
+});
+if (empty($packages)) {
```

Comment:
`$packages` is always an array, so you might consider using a strict equality instead:

 ```suggestion
if ($packages === []) {
```